### PR TITLE
chore: documentation chores, typo, format, links, etc

### PR DESCRIPTION
**Pull request by AkatQuas**
## Summary

Small changes on documentations, typo, symbol sign, links, etc.

To make docs more concise.

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/plugins/rspack/copy-rspack-plugin.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Table } from '@builtIns';
 import WebpackLicense from '@components/webpack-license';
 import { ApiMeta } from '@components/ApiMeta.tsx';
 
-<WebpackLicense from="https://webpack.js.org/plugins/copy-webpack-plugin/#root" />
+{/* <WebpackLicense from="https://webpack.js.org/plugins/copy-webpack-plugin/#root" /> */}
```
Comment (chenjiahan) [inline website/docs/en/plugins/rspack/copy-rspack-plugin.mdx:5]: Is this a mistake?

File: website/docs/zh/api/loader-api/examples.mdx [hunk 2]
```diff
@@ -26,12 +26,12 @@ module.exports = function (content, map, meta) {
 ```
 
 ::: info
-出于技术和性能方面的考虑，Rspack 内部会将所有 loader 转换为异步 loader。
+出于技术和性能方面的考虑， Rspack 内部会将所有 loader 转换为异步 loader。
```
Comment (chenjiahan) [inline website/docs/zh/api/loader-api/examples.mdx:29]: No need to add space after `，`

```suggestion
出于技术和性能方面的考虑，Rspack 内部会将所有 loader 转换为异步 loader。
```

File: website/docs/zh/api/loader-api/examples.mdx [hunk 3]
```diff
@@ -26,12 +26,12 @@ module.exports = function (content, map, meta) {
...
 :::
 
 ## 异步 Loader
 
-对于异步 Loader，`this.async` 被用于获取 `callback` 函数：
+对于异步 Loader ， `this.async` 被用于获取 `callback` 函数：
```
Comment (chenjiahan) [inline website/docs/zh/api/loader-api/examples.mdx:34]:
```suggestion
对于异步 Loader，`this.async` 被用于获取 `callback` 函数：
```

File: website/docs/en/plugins/rspack/copy-rspack-plugin.mdx (hunk 1 above)
Comment (AkatQuas) [reply website/docs/en/plugins/rspack/copy-rspack-plugin.mdx:5]: Source page: https://rspack.dev/plugins/rspack/copy-rspack-plugin

this plugin is `rspack` only, so it seems unnecessary to show a license block

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
Many thanks!

**Pull request merged**